drone_localization/infrastructure/gallery_repository_impl.py

```
@@ -100,14 +121,41 @@ def delete_image(self, image_id: str) -> bool:
+
+            # Затем очищаем MinIO бакет (удаляем все объекты)
+            try:
+                objects = list(self.minio.list_objects(self.bucket, recursive=True))
+                logger.info(f"Found {len(objects)} objects in MinIO bucket '{self.bucket}'")
+
+                if objects:
+                    for obj in objects:
+                        try:
+                            self.minio.remove_object(self.bucket, obj.object_name)
+                            logger.debug(f"Removed object: {obj.object_name}")
+                        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)